### ISSUE TITLE
GS 3.34.1: St.get_slow_down_factor is not a function

### ISSUE THREAD

**Issue by schnittchen** [timbertson/gnome-shell-impatience#18]
From `journalctl /usr/bin/gnome-shell`:

```
Jan 21 22:40:03 rosemary gnome-shell[2641]: JS ERROR: Extension impatience@gfxmonk.net: TypeError: St.get_slow_down_factor is not a function
                                            Ext.prototype._init@/home/thomas/.local/share/gnome-shell/extensions/impatience@gfxmonk.net/extension.js:21:24
                                            Ext@/home/thomas/.local/share/gnome-shell/extensions/impatience@gfxmonk.net/extension.js:14:2
                                            init@/home/thomas/.local/share/gnome-shell/extensions/impatience@gfxmonk.net/extension.js:59:9
                                            _callExtensionInit@resource:///org/gnome/shell/ui/extensionSystem.js:339:34
                                            loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:261:22
                                            callback@resource:///org/gnome/shell/ui/extensionDownloader.js:244:17
                                            gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:108:13
```

**Comment by czenker** [comment]
I ran into the exact the same issue with Gnome 3.34 on Fedora. 

But the issue has been fixed in #16 already. So just update to the [latest version of impatience](https://extensions.gnome.org/extension/277/impatience/) and it will work again.

Thanks @timbertson for keeping the extension up-to-date. It is very useful!

**Comment by schnittchen** [comment]
My bad! I somehow had to restart gnome-shell, now it's working. Thanks @timbertson!
